cpp/tests/casacore.cc

```diff
@@ -81,136 +81,112 @@ class TmpMS : public TmpPath {
   std::unique_ptr<casa::MeasurementSet> ms_;
 };
 
+const std::string test_file = atca_filename("0332-391.ms");
```

Comment:
General comment for this file: maybe add a comment of where the hardcoded numbers are coming from? I assume from this test_file, but is it obvious how to fish them out if someone wants to update this file, for example?